src/main/java/com/vinylteam/vinyl/PropertiesReader.java

```diff
@@ -0,0 +1,55 @@
+package com.vinylteam.vinyl;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+public class PropertiesReader {
+
+    public PropertiesReader() {
+        InputStream inputStream;
+
+        inputStream = getClass().getClassLoader().getResourceAsStream("application.properties");
+
+        setProperties(inputStream);
+    }
+
+    private final Properties properties = new Properties();
+
+    private void setProperties(InputStream inputStream) {
+        try {
+            properties.load(inputStream);
+        } catch (IOException e) {
+            throw new RuntimeException("Was unable to load properties", e);
+        }
+    }
+
+    public String getUser() {
+        return properties.getProperty("user");
+    }
+
+    public String getPassword() {
+        return properties.getProperty("password");
+    }
+
+    public String getDatabaseName() {
+        return properties.getProperty("databaseName");
```

Comment:
why we create getters in PropertiesReader if that is standard method in class Properties - getProperty("some.parameter")

Reply:
My best guess - to limit contact of dbdatasource to only certain properties and not let it fiddle with other properties from properties file. Properties do not contain only DB properties. It's also app properties, and all that.
In order to organize reading specific properties at different layers we have PropertiesReader.
Just a guess, though.